src/commands/transcription.ts

```diff
@@ -0,0 +1,42 @@
+import { ICommandModule, ICommandDefinition, ICommandsMap } from "../types/commands";
+import { Message } from "whatsapp-web.js";
+import { config } from "../config";
+import { TranscriptionMode } from "../types/transcription-mode";
+
+export const TranscriptionModule: ICommandModule = {
+	key: "transcription",
+	register: (): ICommandsMap => {
+		return {
+			enabled,
+			mode
+		} as ICommandsMap;
+	}
+};
+
+const enabled: ICommandDefinition = {
+	help: "<value> - Toggle if transcription is enabled",
+	hint: "true, false",
+	data: config.transcriptionEnabled,
+	execute: function (message: Message, valueStr?: string) {
+		if (["true", "false"].indexOf(valueStr || "") < 0) {
+			message.reply(`Invalid value, please specify true or false`);
+			return;
+		}
+		this.data = valueStr == "true";
```

Comment:
transcriptionEnabled is a boolean. The commands should change the value in the config, then you also dont have to use `getConfig("tts", "enabled")`

Reply:
actually I think we could consider to move all config into commands instead, so every config is overridable in runtime

Reply:
Ok fine, I will review & test this in the next days and merge it. (Quite busy right now)

Thank you for this contribution.